resources/public/api/conf/1.0/common/errors.yaml

```diff
@@ -26,3 +26,9 @@ components:
       value:
         code: MATCHING_RESOURCE_NOT_FOUND
         message: Matching resource not found
+
+    incomeSourceNotFound:
```

Comment:
An "incomeSource" usually related to a business, not a nino. To avoid confusion maybe rename to something that is more specific (to the endpoint in this case since the description references creating checkpoints). E.g. notFoundCreateCheckpoint.